src/libs/PolicyUtils.ts

```diff
@@ -14,10 +14,7 @@ type UnitRate = {rate: number};
  * These are policies that we can use to create reports with in NewDot.
  */
 function getActivePolicies(policies: OnyxCollection<Policy>): Policy[] | undefined {
-    return Object.values(policies ?? {}).filter<Policy>(
-        (policy): policy is Policy =>
-            policy !== null && policy && (policy.isPolicyExpenseChatEnabled || policy.areChatRoomsEnabled) && policy.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE,
-    );
+    return Object.values(policies ?? {}).filter<Policy>((policy): policy is Policy => policy !== null && policy && policy.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE);
```

Comment:
Just want to check that we also meant to remove the `isPolicyExpenseChatEnabled` condition, even though it [remains undeleted](https://github.com/Expensify/App/blob/de0a96b0309c73bb801f19a8e2bddb43dd7e2c75/src/types/onyx/Policy.ts#L65-L66).

Reply:
`policy !== null &&` condition is redundant

Reply:
Yes, given that areChatRoomsEnabled is always true, the other part of the OR is irrelevant

> policy !== null && condition is redundant

Yes, but it was there before plus if I remove it, it complains about the type returned not being valid.
Anyway, since this existed already, can we merge as is?

Reply:
Cool, sounds good